### PR TITLE
OSDOCS-8370: Rosa Getting Started Setup Content Migration

### PR DESCRIPTION
**Pull request by jneczypor**
[OSDOCS-8370](https://issues.redhat.com//browse/OSDOCS-8370): Rosa Getting Started Setup Content Migration from rosaworkshop.io

Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OSDOCS-8370

Link to docs preview:
https://67837--docspreview.netlify.app/openshift-rosa/latest/cloud_experts_tutorials/cloud_experts_getting_started/cloud-experts-getting-started-setup

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jneczypor** [comment]
Hi @yuwang-RH! Can I please get a QE review of this PR? Thanks!

File: _topic_maps/_topic_map_rosa.yml [hunk 1]
```diff
@@ -112,6 +112,12 @@ Topics:
   File: cloud-experts-dynamic-certificate-custom-domain
 - Name: Assigning consistent egress IP for external traffic
   File: cloud-experts-consistent-egress-ip
+- Name: Getting started with ROSA
```
Comment (eohartman) [inline _topic_maps/_topic_map_rosa.yml:115]: Are there going to be more tutorials that are located in the "Getting started with ROSA" subheader? If not, then the Setup page should be located under the Tutorials heading.

File: cloud_experts_tutorials/cloud_experts_getting_started/cloud-experts-getting-started-setup.adoc [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+
+== Installing the ROSA CLI
+. Install the link:https://console.redhat.com/openshift/downloads[ROSA CLI] for your operating system.
+. Download and extract the relevant file for your operating system by using the following command:
++
+[source,terminal]
+----
+tar -xvf rosa-linux.tar.gz
+----
+. Save the file to a location within your path by using the following command:
```
Comment (eohartman) [inline cloud_experts_tutorials/cloud_experts_getting_started/cloud-experts-getting-started-setup.adoc:58]: "your path" sounds weird. Maybe "Save the file to a location with your computer by using the following command."?

File: cloud_experts_tutorials/cloud_experts_getting_started/cloud-experts-getting-started-setup.adoc [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
++
+[source,terminal]
+----
+sudo mv rosa /usr/local/bin/rosa
+----
+. Run `rosa version` to verify successful installation.
```
Comment (eohartman) [inline cloud_experts_tutorials/cloud_experts_getting_started/cloud-experts-getting-started-setup.adoc:64]: "...to verify a successful installation."

File: cloud_experts_tutorials/cloud_experts_getting_started/cloud-experts-getting-started-setup.adoc [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+
+== Installing the OpenShift CLI
+There are a few ways to install the OpenShift CLI (`oc`):
+
+* *Option 1- Using the ROSA CLI:*
```
Comment (eohartman) [inline cloud_experts_tutorials/cloud_experts_getting_started/cloud-experts-getting-started-setup.adoc:69]: I'm not sure if the options should have a - after them or a :  A colon looks better to me, but I have no idea of the rules regarding this. Feel free to ignore!

File: cloud_experts_tutorials/cloud_experts_getting_started/cloud-experts-getting-started-setup.adoc [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+.. Run `rosa download oc`.
+.. Once downloaded, unzip the file and move the executables into a directory in your path.
```
Comment (eohartman) [inline cloud_experts_tutorials/cloud_experts_getting_started/cloud-experts-getting-started-setup.adoc:71]: Another "your path."

File: cloud_experts_tutorials/cloud_experts_getting_started/cloud-experts-getting-started-setup.adoc [hunk 6]
```diff
@@ -0,0 +1,227 @@
...
++--------------+----------------------------------------+--------------------+
+----
+
+== Ensuring the ELB service role exists
+[TIP]
+====
+Make sure that the service role for the ELB already exists, otherwise cluster deployment could fail.
+====
+
+Run the following command to check for the ELB service role and create it if it is missing:
```
Comment (eohartman) [inline cloud_experts_tutorials/cloud_experts_getting_started/cloud-experts-getting-started-setup.adoc:134]: I think these steps should be numbered, similar to the "Verifying the configuration" section.

File: cloud_experts_tutorials/cloud_experts_getting_started/cloud-experts-getting-started-setup.adoc [hunk 7]
```diff
@@ -0,0 +1,232 @@
...
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2023-11-13
+
+There are currently two supported credential methods when creating a {product-title} (ROSA) cluster. One method uses an IAM user with the `AdministratorAccess` policy. The second and *recommended* method uses Amazon Web Services (AWS) Security Token Service (STS). 
+//To be added when the ROSA with STS Explained tutorial is published:
+//For more information, see the xref../cloud_experts_tutorials/cloud_experts_rosa_with_sts_explained.adoc#id[ROSA with STS Explained] tutorial. This workshop uses the STS method.
+
+== Prerequisites
+
+Please review the prerequisites listed in the xref:../../rosa_planning/rosa-cloud-expert-prereq-checklist.adoc#rosa-cloud-expert-prereq-checklist[Prerequisites for ROSA with STS] checklist.
```
Comment (bmcelvee) [inline cloud_experts_tutorials/cloud_experts_getting_started/cloud-experts-getting-started-setup.adoc:18]: Suggestion: 
s/Please review/review

File: cloud_experts_tutorials/cloud_experts_getting_started/cloud-experts-getting-started-setup.adoc [hunk 8]
```diff
@@ -0,0 +1,232 @@
...
+
+== Enabling ROSA
+
+[NOTE]
+====
+Only complete this step if you have *not* enabled ROSA in your AWS account.
+====
+
+. Visit the link:https://console.aws.amazon.com/rosa[AWS console] to enable your account to use ROSA.
+. Click on the orange *Enable OpenShift* button.
```
Comment (bmcelvee) [inline cloud_experts_tutorials/cloud_experts_getting_started/cloud-experts-getting-started-setup.adoc:42]: Suggestion: 
s/Click on/Click

File: cloud_experts_tutorials/cloud_experts_getting_started/cloud-experts-getting-started-setup.adoc [hunk 9]
```diff
@@ -0,0 +1,232 @@
...
+[source,terminal]
+----
+aws iam get-role --role-name "AWSServiceRoleForElasticLoadBalancing" || aws iam create-service-linked-role --aws-service-name "elasticloadbalancing.amazonaws.com"
+----
+
+== Logging in to your Red Hat account
+. Enter `rosa login` in a terminal.
+. It will prompt you to open a web browser and go to the link:https://console.redhat.com/openshift/token/rosa[Red Hat console].
+. Log in, if necessary.
+. Click on *Load token*.
```
Comment (bmcelvee) [inline cloud_experts_tutorials/cloud_experts_getting_started/cloud-experts-getting-started-setup.adoc:163]: Suggestion: 
s/Click on/Click

**Pull request merged**